linker/shared/src/main/scala/org/scalajs/linker/analyzer/Analyzer.scala

```diff
@@ -985,31 +995,30 @@ private final class Analyzer(config: CommonPhaseConfig,
 
   private def createMissingMethodInfo(
       encodedName: String): Infos.MethodInfo = {
-    makeSyntheticMethodInfo(encodedName = encodedName)
+    makeSyntheticMethodInfo(encodedName)
   }
 
-  private def createMissingStaticMethodInfo(
+  private def createMissingStaticLikeMethodInfo(namespace: MemberNamespace,
       encodedName: String): Infos.MethodInfo = {
-    makeSyntheticMethodInfo(encodedName = encodedName, isStatic = true)
+    makeSyntheticMethodInfo(encodedName, namespace)
   }
```

Comment:
I feel like these two helpers are not warranted. In total, they are used 3 times and can be replaced by `makeSyntheticMethodInfo`.

Note that one call to `createMissingMethodInfo` seems wrong, as it refers to a constructor.